packages/miniflare/test/fixtures/assets/modules.ts

```diff
@@ -0,0 +1,35 @@
+import manifestJSON from "__STATIC_CONTENT_MANIFEST";
```

Comment:
I don't think we should support importing this manifest. It should only be available on the `env` object. This was only here for back compatibility with how Workers Sites worked.

Reply:
indeed. The whole manifest thing kinda had me very confused. I hadn't realised this was specific to Workers Sites and the `kv-asset-handler`. Removed!